src/DssSpell.sol

```diff
@@ -48,36 +151,275 @@ contract DssSpellAction is DssAction {
+    address internal constant USDS_IMP                     = 0x1923DfeE706A8E78157416C29cBCCFDe7cdF4102;
+    address internal constant USDS_JOIN                    = 0x3C0f895007CA717Aa01c8693e59DF1e8C3777FEB;
+    address internal constant DAI_USDS                     = 0x3225737a9Bbb6473CB4a45b7244ACa2BeFdB276A;
+    address internal constant SUSDS                        = 0xa3931d71877C0E7a3148CB7Eb4463524FEc27fbD;
+    address internal constant SUSDS_IMP                    = 0x4e7991e5C547ce825BdEb665EE14a3274f9F61e0;
+    address internal constant SKY                          = 0x56072C95FAA701256059aa122697B133aDEd9279;
+    address internal constant MKR_SKY                      = 0xBDcFCA946b6CDd965f99a839e4435Bcdc1bc470B;
+    address internal constant PAIR_DAI_MKR                 = 0x517F9dD285e75b599234F7221227339478d0FcC8;
+    address internal constant PAIR_USDS_SKY                = 0x2621CC0B3F3c079c1Db0E80794AA24976F0b9e3c;
+    address internal constant MCD_SPLIT                    = 0xBF7111F13386d23cb2Fba5A538107A73f6872bCF;
+    address internal constant SPLITTER_MOM                 = 0xF51a075d468dE7dE3599C1Dc47F5C42d02C9230e;
+    address internal constant MCD_FLAP                     = 0xc5A9CaeBA70D6974cBDFb28120C3611Dd9910355;
+    address internal constant FLAP_SKY_ORACLE              = 0x38e8c1D443f546Dc014D7756ec63116161CB7B25;
+    address internal constant MCD_VEST_SKY                 = 0xB313Eab3FdE99B2bB4bA9750C2DDFBe2729d1cE9;
+    address internal constant REWARDS_USDS_SKY             = 0x0650CAF159C5A49f711e8169D4336ECB9b950275;
+    address internal constant REWARDS_DIST_USDS_SKY        = 0x2F0C88e935Db5A60DDA73b0B4EAEef55883896d9;
+    address internal constant REWARDS_USDS_01              = 0x10ab606B067C9C461d8893c47C7512472E19e2Ce;
+    address internal constant CRON_REWARDS_DIST_JOB        = 0x6464C34A02DD155dd0c630CE233DD6e21C24F9A5;
+    address internal constant WRAPPER_USDS_LITE_PSM_USDC_A = 0xA188EEC8F81263234dA3622A406892F3D630f98c;
+
+    // ---------- MCD Addresses ----------
+    address internal MCD_PAUSE = DssExecLib.getChangelogAddress("MCD_PAUSE");
 
     function actions() public override {
-        // ---------- Launch Project Funding ----------
-        // Forum: https://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/22
-        // MIP: https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget
 
-        // Launch Project - 9535993 DAI - 0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F
-        DssExecLib.sendPaymentFromSurplusBuffer(LAUNCH_PROJECT_FUNDING, 9_535_993);
+        // Note: load the MCD contracts depencencies
+        DssInstance memory dss = MCD.loadFromChainlog(DssExecLib.LOG);
+
+        // ---------- New Tokens Init ----------
+        // Forum: TODO
+        // Poll: TODO
+        // MIP: TODO
+
+        // Init USDS by calling UsdsInit.init with the following parameters:
```

Comment:
There are a bunch of instructions now explains what init does under the hood ([e.g.](https://docs.google.com/spreadsheets/d/1w_z5WpqxzwreCcaveB2Ye1PP5B8QAHDglzyxKHG3CHw/edit?gid=1165645248#gid=1165645248&range=B38:B41)). I suggest to add them under the call and explain each of them via `// Note: this action is done inside UsdsInit.init function`

Reply:
I see the exec sheet was updated with this "hidden" actions, so I'll just copy them over preserving the order, with no `Note:` label.